azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,325 @@
+# Azure Cognitive Services Computer Vision SDK for Python
+
+The Computer Vision service provides developers with access to advanced algorithms for processing images and returning information. Computer Vision algorithms analyze the content of an image in different ways, depending on the visual features you're interested in. For example, Computer Vision can determine if an image contains adult or racy content, find all the faces in an image, get handwritten or printed text. This service works with popular image formats, such as JPEG and PNG. 
+
+You can use Computer Vision in your application to:
+
+- Analyze images for insight
+- Extract text from images
+- Generate thumbnails
+
+Looking for more documentation?
+
+* [SDK reference documentation](https://docs.microsoft.com/python/api/azure-cognitiveservices-vision-computervision/azure.cognitiveservices.vision.computervision)
+* [Cognitive Services Computer Vision documentation](https://docs.microsoft.com/azure/cognitive-services/computer-vision/)
+
+## Prerequisites
+
+* Azure subscription - [Create a free account][azure_sub]
+* Azure [Computer Vision resource][computervision_resource]
+* [Python 3.6+][python]
+
+If you need a Computer Vision API account, you can create one with this [Azure CLI][azure_cli] command:
+
+```Bash
+RES_REGION=westeurope 
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+az cognitiveservices account create \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --location $RES_REGION \
+    --kind ComputerVision \
+    --sku S1 \
+    --yes
+```
+
+## Installation
+
+Install the Azure Cognitive Services Computer Vision SDK with [pip][pip], optionally within a [virtual environment][venv].
+
+### Configure a virtual environment (optional)
+
+Although not required, you can keep your base system and Azure SDK environments isolated from one another if you use a [virtual environment][virtualenv]. Execute the following commands to configure and then enter a virtual environment with [venv][venv], such as `cogsrv-vision-env`:
+
+```Bash
+python3 -m venv cogsrv-vision-env
+source cogsrv-vision-env/bin/activate
+```
+
+### Install the SDK
+
+Install the Azure Cognitive Services Computer Vision SDK for Python [pip package][pypi_computervision] with [pip][pip]:
```

Comment:
`pip` is the tool one uses to install _distribution packages_; I've not seen them commonly referred to as "pip packages." Here's a little blurb on the terminology:

https://packaging.python.org/glossary/#term-distribution-package

They're generally referred to simply as _packages_, so maybe something like:

`Install the Azure Cognitive Services Computer Vision SDK for Python [package from PyPi][pypi_computervision] with [pip][pip]:`

(Full disclosure: I had to look this up.) :wink:

Reply:
I asked several python devs and they said just "package". Is that ok with you?

Reply:
Yup, that's my recommendation.